src/renderers/webgl/WebGLRenderStates.js

```diff
@@ -101,7 +101,13 @@ function WebGLRenderStates() {
 
 	function dispose() {
 
-		renderStates = new WeakMap();
+		renderStates.forEach( function ( scene ) {
```

Comment:
I was going fast, `scene` is actually an `id` number here. We need a way to get the scenes, to remove the handlers.

Reply:
Nevermind, I was thinking of some other code (porting Three to AssemblyScript) where I currently have id numbers there.